flow/connectors/mongo/mongo.go

```diff
@@ -371,7 +395,7 @@ func defaultPipeline() mongo.Pipeline {
 	return mongo.Pipeline{
 		{{Key: "$match", Value: bson.D{
 			{Key: "operationType", Value: bson.D{
-				{Key: "$in", Value: bson.A{"insert", "update", "replace", "delete"}},
+				{Key: "$in", Value: bson.A{"insert", "update", "replace", "delete", "invalidate"}},
```

Comment:
nit: is this intentionally added as an assertion?  (since deployment level change stream doesn't get invalidate)

Reply:
Going to delete for now, can be introduced separately if needed.  the data contains different fields than the other operationType so would encounter deserialization issue today since it's not handled.